Content.Client/Administration/UI/Tabs/ObjectsTab/ObjectsTab.xaml.cs

```diff
@@ -40,38 +37,25 @@ public ObjectsTab()
 
         foreach (var type in Enum.GetValues(typeof(ObjectsTabSelection)))
```

Comment:
```suggestion
        foreach (var type in Enum.GetValues<ObjectsTabSelection>())
```
New generics allow defining type so that it doesn't need a cast in the loop.